site/hugo.toml

```diff
@@ -107,11 +107,11 @@ archived_version = false
 # The version number for the version of the docs represented in this doc set.
 # Used in the "version-banner" partial to display a version number for the
 # current doc set.
-version = "latest"
```

Comment:
would `latest` still be available ?

Reply:
sure, this's some parmas we use in template/partials/layouts